### ISSUE TITLE
Add hotkeys for better keyboard navigation

### ISSUE THREAD

**Issue by theilm** [neos/neos-ui#1594]
### Description

In order to make keyboard navigation easier and improve accessibility, keyboard shortcuts should be added to central components (e.g. for opening the inspector or for publishing changes).

**Comment by Benjamin-K** [comment]
In the current version you can use some keyboard shortcuts. But now there are some other problems:

When you edit a text field and type one of the shortcuts (for example "t e" which is in many words) the shortcut is triggered and the focus for the current element gets lost. This should be fixed.

**Comment by KPeschke** [comment]
We've the same problems as Benjamin-K.

How can we deactivate the hotkeys? The Button ist still available in backend.

**Comment by theilm** [comment]
@KPeschke @Benjamin-K if you have problems with hotkeys, you can disable them via configuration:
https://github.com/neos/neos-ui/blob/master/Configuration/Settings.yaml#L56

**Comment by KPeschke** [comment]
@theilm Thanks for your reply.

I have tried to disable the hotkeys, but the button in backend is still visible.

```
Neos:
  Neos:
    Ui:
      frontendConfiguration:
        hotkeys: null
```

Also tried with `false` and `[]` alwas the same result.

![image](https://user-images.githubusercontent.com/8969952/42636196-6ed08718-85e8-11e8-8fc5-9fb06ad9f88d.png)

**Comment by mstruebing** [comment]
Should be fixed by #2016
